elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -981,6 +981,8 @@ elytron.distributed-realm.remove=The remove operation for the security realm.
 # Attributes
 elytron.distributed-realm.realms=References to one or more security realms for authentication and authorization.
 elytron.distributed-realm.realms.realm=Name of the referenced realm.
+elytron.distributed-realm.ignore-unavailable-realms=Allow users to specify that the search should continue on to the next realm if a realm happens to be unavailable.
```

Comment:
Maybe this could be simplified to:

Whether subsequent realms should be checked after an unavailable realm is reached. The default value is false.